adapters/map.go

```diff
@@ -0,0 +1,28 @@
+package adapters
+
+import (
+	"github.com/smartcontractkit/chainlink/store"
+	"github.com/smartcontractkit/chainlink/store/models"
+)
+
+// Map holds holds a path to the desired field in the `data` JSON object,
+// made up of an array of strings.
+type Map struct {
+	Path []string `json:"path"`
+}
+
+// Perform returns the value from a desired mapping within the `data` JSON object,
+// this is specifically used for bridge adaptor returns
+//
+// For reference on how the parsing is done, refer to JsonParse
+func (m *Map) Perform(input models.RunResult, store *store.Store) models.RunResult {
+	jp := JSONParse{Path: m.Path}
+
+	data, err := input.Data.Add("value", input.Data.String())
```

Comment:
I think this would be more reusable if we allowed you to pass any key and specify what key to move it to, so that "value" was not hardcoded.

For example: `{"type": "copy", "fromHere":"toHere"}` would copy the value at `fromHere` and place it in the key `toHere`.

Also, I'm inclined to just take a string and not an array, so that we don't have to deal with nesting objects. We may need that, but I'd like to wait until more use cases for it pop up before we design for it.